demos/index.html

```diff
@@ -0,0 +1,116 @@
+            </section>
+
+            <section class="gpiic-adjusterContainer-contrast gpii-adjusterContainer-contrast gpii-main">
+                <!-- ---- begin Contrast adjuster template ---- -->
+                <p class="gpii-instructions">You can choose a screen colour to make things easier to see.</p>
+                <div class="gpii-controlsContainer small-offset-2 medium-offset-3 small-7 medium-6 end text-center">
+                    <div class="gpii-choice">
+                        <input class="flc-prefsEditor-theme gpii-hidden-accessible" name="theme" id="theme-default" value="default" type="radio" checked>
+                        <span class="gpii-icon gpii-indicator"></span>
+                        <label for="theme-yes" class="flc-prefsEditor-theme-label gpii-theme gpii-theme-default" id="theme-yes-label">no change</label>
+                    </div>
+                    <div class="gpii-choice">
+                        <input class="flc-prefsEditor-theme gpii-hidden-accessible" name="theme" id="theme-bw" value="bw" type="radio">
+                        <span class="gpii-icon gpii-indicator"></span>
+                        <label for="theme-bw" class="flc-prefsEditor-theme-label gpii-theme gpii-theme-bw" id="theme-bw-label">black on white</label>
+                    </div>
+                    <div class="gpii-choice">
+                        <input class="flc-prefsEditor-theme gpii-hidden-accessible" name="theme" id="theme-wb" value="wb" type="radio">
+                        <span class="gpii-icon gpii-indicator"></span>
+                        <label for="theme-wb" class="flc-prefsEditor-theme-label gpii-theme gpii-theme-wb" id="theme-no-label">white on black</label>
+                    </div>
+                </div>
+                <!-- ---- end Contrast adjuster template ---- -->
+            </section>
+
+            <nav class="gpii-nav">
+                <div class="gpii-backNext">
+                    <button class="left gpii-back" tabindex="3" ><span class="gpii-icon"></span>back</button>
+                    <button class="right gpii-next" tabindex="4" >next<span class="gpii-icon"></span></button>
+                </div>
+                <div class="text-center gpii-navIcons">
+                    <ul>
+                        <li class="gpii-navIcon gpii-lang">
+                            <div class="gpii-activeIndicator gpii-hidden"></div>
+                            <div class="gpii-doneIndicator"></div>
+                        </li>
+                        <li class="gpii-navIcon gpii-volume">
+                            <div class="gpii-activeIndicator gpii-hidden"></div>
+                            <div class="gpii-doneIndicator"></div>
+                        </li>
+                        <li class="gpii-navIcon gpii-text gpii-active">
+                            <div class="gpii-activeIndicator gpii-show"></div>
+                            <div class="gpii-doneIndicator gpii-show"></div>
```

Comment:
I know this will be adjusted programmatically so it doesn't really matter, but I wonder if we could "show" the 'done' indicator on the first two icons and only the 'active' indicator on the current one?